main.go

```diff
@@ -80,6 +80,8 @@ func main() {
 				log.Fatal(err)
 			}
 
+			keyring.Set(name, "token", tkn)
```

Comment:
Can you goroutinize this?

```suggestion
			go keyring.Set(name, "token", tkn)
```

Reply:
It could be my system, but I had that initially and it never set the token, and also froze the UI

Reply:
Possibly- it's what happened in #165, I remember facing it too

Reply:
This is not the reason for issue #165, it has something to do with captcha.

Reply:
Okay, my bad. After a few more attempts it seems it was just logging in that was taking really long, but it was coincidentally faster with this change at some point.